### ISSUE TITLE
Get credentials issued in datetime window

### ISSUE THREAD

**Issue by OR13** [w3c-ccg/vc-http-api#114]
It's useful to be able to ask a vendor for all credentials issued in a date time window.

I propose we add optional get interfaces that expose credentials over http.

**Comment by mprorock** [comment]
Thumbs up from us.  Generally a GET/PUT/POST/DELETE and other expected REST collection methods for core objects such as credentials as being dealt with by the api.

**Comment by mprorock** [comment]
> We may also want to support more advanced querying, by json object property matching.

There are some good models for this that we can reference related to mongo, snowflake and others

**Comment by OR13** [comment]
Yes, i think we would like some simple date range queries for `created` `issuanceDate` `expirationDate`... and startsWith, endsWith for string properties in an object.... this will allow for external systems to use their existing identifier such as GLNs or LEIs to query credentials... in the same way... for all vendors :)
